docs/benchmarks/micro-benchmarks.md

```diff
@@ -26,18 +26,14 @@ id: micro-benchmarks
         <b>Micro Benchmark</b>
         <img src={require('../assets/bar.png').default}/>
       </td>
-      <td>
-        <b>Model Benchmark</b>
-        <img src={require('../assets/bar.png').default}/>
```

Comment:
remove the unused image?

Reply:
I will remove it since <img src={require('../assets/bar.png').default}/> is not work. It cannot be showed on GitHub

Reply:
removed the image